### PR TITLE
Update the CI chain

### PR DESCRIPTION
**Pull request by leofeyer**
This is not going to be fun. 😄

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -168,13 +159,13 @@ jobs:
...
         name: Require Checker
-        runs-on: ubuntu-18.04
+        runs-on: ubuntu-latest
         if: github.event_name == 'pull_request'
         steps:
             - name: Setup PHP
               uses: shivammathur/setup-php@v2
               with:
-                  php-version: 8.1
+                  php-version: 8.0
```
Comment (m-vo) [inline .github/workflows/ci.yml]: Intentional downgrade?

**Comment by leofeyer** [comment]
Nice, it actually went smooth. Seems the MySQL issues in `ubuntu-latest` have been resolved meanwhile. 🎉 

Only Codecov still does not like PHP 8 (also mentioned in #4051).

**Comment by leofeyer** [comment]
Could be this: 😄 

```
PHPUnit 8.5.23 by Sebastian Bergmann and contributors.
Error: This version of PHPUnit does not support code coverage on PHP 8
```

**Comment by m-vo** [comment]
:see_no_evil: We should probably upgrade to 9.3 anyways… 

I ran into PHP8 related issues with 8.5, too, when adjusting the attributes implementation. And there will probably more in the future.

**Review by m-vo — APPROVED** [review]

**Pull request merged**